README.md

```diff
@@ -42,19 +44,33 @@ iframe.src = vfrLink
 
 ### Iframe settings
 
-Add query parameters to the link to configure the optional settings:
+Add query parameters to the link to configure the component behavior. Virtual Fitting Room and 3D Viewer parameters differ slightly.
 
-* `modelid` — a comma-separated list of model identifiers for the models that should be displayed for try-on.
-* `startwithid` — the identifier of the model that should be loaded first. Note that the order of the models won't change.
-* `locale` — the locale code for the Virtual Fitting Room UI language and metadata. If the specified locale isn't available, the English locale will be loaded instead.
+| Parameter               | Fitting Room      | 3D Viewer                 | Description                                                                                      |
+|-------------------------| :---------------: | :-----------------------: |--------------------------------------------------------------------------------------------------|
+| `modelid`, **required** | **+**<br />a list | **+**<br />a single value | The model identifiers for the models that should be displayed in the component.                                   |
```

Comment:
modelid is mandatory for a 3D Viewer. We need to add this notice

Reply:
It says **required** in the first column

Reply:
It is required only for a 3D Viewer setup. For VTO this parameter is optional.